content/en/agent/docker/_index.md

```diff
@@ -133,6 +134,8 @@ Learn more about [DogStatsD over Unix Domain Sockets][15].
 
 ### Tagging
 
+**Note**: As a best practice, Datadog recommends using [unified service tagging](#unified-service-tagging) when assigning tags.
```

Comment:
Starting the paragraph with **Note** can reduce the readability; suggesting to remove and simply continue with the rest of the sentence/